src/Serilog.Sinks.DiagnosticTrace/project.json

```diff
@@ -0,0 +1,30 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The diagnostic trace sink for Serilog - Simple .NET logging with fully-structured events",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "trace", "diagnostic" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": { "target": "project" }
+  },
+  "compilationOptions": {
+    "keyFile": "../../assets/Serilog.snk"
+  },
+  "frameworks": {
+    "net45": {
+    },
+    "dotnet5.1": {
+      "dependencies": {
+        "System.Diagnostics.TraceSource": "4.0.0-beta-23516"
+      }
+    },
+    "dotnet5.4": {
+      "dependencies": {
+        "System.Diagnostics.Process": "4.1.0-beta-23516",
+        "System.Diagnostics.TraceSource": "4.0.0-beta-23516"
```

Comment:
Same here. `System.Diagnostics.TraceSource` targets `dotnet5.1`. What's the reason why this can't be moved up and this target removed? Isn't this used in `dotnet5.1`?